### PR TITLE
Setup mobile first

### PR DESCRIPTION
**Pull request by AbdulSalamAbro**
*Added Header Section
*Heading section
*Paragraph section
*Social Icons
*Fixed Linters Errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        <div class="container">
+            <h1 >
+                <span id="name-heading">
+                    Hey there<br>
+                I'm Abdul Salam
+                </span>
+                <span id="job-role">
+                    I',m a Software<br>
+                    Developer
+                </span>
```
Comment (Munsa1) [inline index.html:29]: - [ ] The main title of the headline section should be wrapped in an `h1` tag for proper HTML page structure reasons, Kindly fix this.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+           
+              <section id="social-icons">
+                <ul>
+                    <li><a href="https://twitter.com/Abdul_SalamAbro"><img src="assets/icons/twitter.png" alt=""></a></li>
+                    <li><a href="https://www.linkedin.com/in/abdul-salam-abro/"><img src="assets/icons/linkedin.png" alt=""></a></li>
+                    <li><a href="https://medium.com/abdulsalam"><img src="assets/icons/medium.png" alt=""></a></li>
+                    <li><a href="https://github.com/AbdulSalamAbro"><img src="assets/icons/github.png" alt=""></a></li>
+                    <li><a href="https://venture.angellist.com/v/Abdulsalam"><img src="assets/icons/angellist.png" alt=""></a></li>  
+                </ul>
+            </section>
```
Comment (Munsa1) [inline index.html]: - [ ] (Optional) I would advise you to wrap the social media links in a `div` as the main container and not the `section` tag as you have done. As a rule, a section wraps an entire block, in this case, the introductory page. The elements inside this section should be `divs`, `articles`, `paragraphs` etc. according to what they hold

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @AbdulSalamAbro, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Responsive Design :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._
### Important 🔴 
Due to insufficient files changed, I will leave some comments here

- [ ] Kindly remove the link to the `Live Demo` section as you have removed it and so this link is broken and goes nowhere
image

- [ ] Kindly fix the broken link to your MIT License agreement, when the user clicks on the link it lands them on a 404 page
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (AbdulSalamAbro) [reply index.html:29]: Hi you can see the code it is already wrapped with h1 tag but due to different color of text in heading I use span to apply different colors and styles

**Review by amejid — APPROVED** [review]
Hi @AbdulSalamAbro, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Great UI design :heavy_check_mark: 
- Changes implemented :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**